hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHA.java

```diff
@@ -726,7 +726,7 @@ public void testOMProxyProviderFailoverToCurrentLeader() throws Exception {
     // information must be returned via the response and a failover must
     // happen to the leader proxy node.
     createVolumeTest(true);
-    Thread.sleep(2000);
+    Thread.sleep(5000);
```

Comment:
Can we use lambdaTestUtils instead of sleep?
```suggestion
    LambdaTestUtils.await(6000, 2000, () -> (omFailoverProxyProvider.getCurrentProxyOMNodeId().equals(leaderOMNodeId)));
```

Reply:
Didn't realize till Bharat pointed out that this test is disabled. I ll revert the change to test and we can fix this when enabling TestOzoneManagerHA. That test was disabled as it was flaky and would need more work before it can be enabled.

Reply:
Sounds good!